requirements.txt

```diff
@@ -1,15 +1,8 @@
 # This file is also processed by conda-build, so keep spaces as "pkg >=1.2"
 appdirs
-cloudpickle
-dask >=0.17.0
-holoviews
+dask[bag] >=1.0
 jinja2
-msgpack-numpy
-msgpack-python
```

Comment:
Was this dropped intentionally? Will it be installed an indirect dependency? I can't find it as a dependency of either msgpack-numpy or dask.

Reply:
It's just `msgpack` in pypi... I've added back `msgpack-numpy` in the dataframe section, and also made it optional. Note that the conda dependencies have not changed at all yet.